.circleci/config.yml

```diff
@@ -13,11 +13,32 @@ e2eTest: &e2eTest
           wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2ectl/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
           chmod +x ./e2ectl
 
+    - run:
+        name: Get kubectl binary
+        command: |
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+          chmod +x ./kubectl
+
+    - run:
+        name: Get helm binary
+        command: |
+          curl -LO https://get.helm.sh/helm-v2.12.0-linux-amd64.tar.gz
+          tar xzvf helm-v2.12.0-linux-amd64.tar.gz
+          mv linux-amd64/helm .
+          chmod +x helm
+          rm -rf linux-amd64/
+
     - run:
         name: Init cluster
         command: |
           ./e2ectl cluster create
           cp $(./e2ectl kubeconfig path) ${E2E_TEST_DIR}/kubeconfig
+          if [ -n "$HELM_INIT" ]
+          then
+            export KUBECONFIG=$(./e2ectl kubeconfig path)
```

Comment:
Instead of creating helm by codes in e2etest, it's better to install helm from outside of it.